build/release.js

```diff
@@ -37,6 +37,7 @@ Release.define({
 	generateArtifacts: function( done ) {
 		Release.exec( "grunt release", "Grunt command failed" );
 		done([
```

Comment:
sorry for beeing late to the party:

please add a note like `keeps this is of files in sync with package.json "files"-key`

Reply:
Done!